compiler/qsc_frontend/src/compile.rs

```diff
@@ -514,8 +514,14 @@ fn resolve_all(
         dropped_names.extend(unit.dropped_names.iter().cloned());
     }
 
+    // bind all symbols in `add_local_package`
     let mut errors = globals.add_local_package(assigner, package);
     let mut resolver = Resolver::new(globals, dropped_names);
+
+    // bind all exported symbols in a follow-on step
+    resolver.resolve_exports(package);
```

Comment:
I think the behavior is ok here, but the method name and comment confused me.

Would "resolve and bind all imports and exports" be a more complete description of what's happening here?